src/content/docs/apm/new-relic-apm/getting-started/introduction-apm.mdx

```diff
@@ -16,150 +16,172 @@ redirects:
+
+import javalogo from 'images/javalogo.png'
+
+import dotnetlogo from 'images/dotnetlogo.png'
+
+import nodejslogo from 'images/nodejslogo.png'
+
+import pythonlogo from 'images/pythonlogo.png'
+
+import phplogo from 'images/phplogo.png'
+
+import rubylogo from 'images/rubylogo.png'
+
+New Relics' application performance monitoring (APM) provides a unified monitoring service for all your apps and microservices. Monitor everything from the hundreds of dependencies of a modern stack down to simple web-transaction times and throughput of an app. Keep track your app's health in real-time by monitoring your **metrics**, **events**, **logs**, and **transactions** (MELT) through pre-built and custom dashboards. 
+
+Our APM service provides the flexability to monitor the exact things you need from your app by automatically instrumenting your code when you install one of our agents. 
+
+- Is your early-access software still slightly unstable? Proactively monitor and solve those errors before they affect your users with errors inbox.  
+- What if your users always comment on how speedy your webapp is and you want to quantify that feedback? Measure their satisfaction by monitoring your Apdex in a glance.
+- Need somewhere to store your logs? Our agents automatically ingest them. 
+- What if your modern stack has dozens or even hundreds of dependencies to keep track of? Easily track them with automap and external services. 
+
+Want to save time with a single unified monitoring solution? Click a logo to get started with APM. It takes just a few minutes!
+
+
+<TechTileGrid>
+  <TechTile
+    name="C SDK"
+    icon="logo-c"
+    to="https://one.newrelic.com/nr1-core?state=af6becca-be1e-7436-bfaf-6d953ecb6d17"
+  />
+
+  <TechTile
+    name="Go agent"
+    icon="logo-go"
+    to="https://one.newrelic.com/nr1-core?state=0545168b-2b10-a80e-e884-e4012df58a54"
+  />
+
+  <TechTile
+    name="Java agent"
+    icon="logo-java"
+    to="https://one.newrelic.com/nr1-core?state=c64efd1c-9107-c4d0-21c0-2b1d859a2231"
+  />
+
+  <TechTile
+    name=".NET agent"
+    icon="logo-dotnet"
+    to="https://one.newrelic.com/nr1-core?state=abd2b6e8-1742-cabf-ffab-f3bba2804149"
+  />
+
+  <TechTile
+    name="Node.js agent"
+    icon="logo-nodejs"
+    to="https://one.newrelic.com/nr1-core?state=65cc8ebd-2c1f-2756-44c4-aed0a37d67c8"
+  />
+
+  <TechTile
+    name="PHP agent"
+    icon="logo-php"
+    to="https://one.newrelic.com/nr1-core?state=ea8c1b2b-a8ab-b9e7-13b7-5f8db05848ff"
+  />
+
+  <TechTile
+    name="Python agent"
+    icon="logo-python"
+    to="https://one.newrelic.com/nr1-core?state=754ab217-ad1d-a3d6-22a5-8d354c7709a5"
+  />
+
+  <TechTile
+    name="Ruby agent"
+    icon="logo-ruby"
+    to="https://one.newrelic.com/nr1-core?state=bca86cc6-7cca-88e9-d4d6-1a3b7e0c1b78"
+  />
+
+</TechTileGrid>
 
 <img
-  title="APM example"
+  title="APM overview"
   alt="An image showing an example APM page."
-  src={apmScreenshot}
+  src={apmOverview}
```

Comment:
In Gatsby preview it looks a bit odd to have this screenshot immediately after the tiles. The tile images are meant to be clicked, the screenshot is not meant to be clicked. Our layout doesn't allow for enough space to make these distinctly separate. I'd either remove the tiles here, or move the screenshot somewhere else.